src/generators/src/infra/database/migrations/migrations.js

```diff
@@ -60,11 +81,14 @@ module.exports =
         const migrationFullPath = path.normalize(`${migrationsPath}/${migrationName}_${camelCase(name)}s.js`)
 
         const columns = createColumns(schema)
+        const ref = identifyRef(schema)
+        const idColumn = Object.values(schema).find(column => column.options.isId)?.name
+
 
         await generate({
           template: `infra/data/database/${db.toLowerCase()}/migration.ejs`,
           target: migrationFullPath,
-          props: { table: `${camelCase(name)}s`, columns: columns }
+          props: { table: `${camelCase(name)}s`, externalColumnName: `${camelCase(name)}Id`, columns, ref, idColumn }
```

Comment:
Both table and columns name should be snake_cases for DB to keep with the current standard. 

ex: `userId` -> `user_id`